pydoctor/epydoc/markup/epytext.py

```diff
@@ -1263,19 +1266,41 @@ def parse_docstring(docstring: str, errors: List[ParseError]) -> ParsedDocstring
                     cast(str, cast(Element, field.children.pop(0)).children[0])
             else:
                 arg = None
+            
+            # TODO: Use ParsedTypeDocstring for numpy and google by default and
+            # allow the other markup to use it as well with a CLI option like --process-types
 
             # Process the field.
             field.tag = 'epytext'
-            fieldDoc = ParsedEpytextDocstring(field, ())
+
+            field_parsed_doc: ParsedDocstring = ParsedEpytextDocstring(field, ())
+
             lineno = int(field.attribs['lineno'])
-            fields.append(Field(tag, arg, fieldDoc, lineno))
+            
+            # This allows epytext markup to use TypeDocstring as well with a CLI option: --process-types
+            if processtypes and tag in ['type', 'rtype']:
+                field_parsed_doc = ParsedTypeDocstring(field_parsed_doc.to_node(), lineno=lineno)
+                for warning_msg in field_parsed_doc.warnings:
+                    errors.append(ParseError(warning_msg, lineno, is_fatal=False))
+            
+            fields.append(Field(tag, arg, field_parsed_doc, lineno))
 
     # Save the remaining docstring as the description.
     if tree_children and tree_children[0].children:
         return ParsedEpytextDocstring(tree, fields)
     else:
         return ParsedEpytextDocstring(None, fields)
 
+def get_parser(obj: Optional[Documentable]) -> Callable[[str, List[ParseError], bool], ParsedDocstring]:
+    """
+    Get the L{parse_docstring} function. 
+    """
+    return parse_docstring
+
+def set_nodes_parent(nodes: Iterable[nodes.Node], parent: nodes.Node) -> Iterable[None]:
+    for node in nodes:
+        node.parent = parent
+        yield node
```

Comment:
This function is offered by the `epydoc.docutils` module, so it should be removed from here.